### ISSUE TITLE
Tabs example possibly incorrect

### ISSUE THREAD

**Issue by matildepark** [markdoc/markdoc#71]
After implementing the Tabs code, I keep hitting `children.map is not a function` on the React renderer output in a Next.js client-side deployment.

<img width="990" alt="image" src="https://user-images.githubusercontent.com/20846414/171510798-03d3bd75-a5e4-4167-95f1-6c5cfcaab39e.png">

Code is basically verbatim off the site, and removing the tags makes the page work again; it's just when I use the tabs (even the example Markdown-side tabs from the site) that the page can't render.

```js
const tab = {
  render: "Tab",
  attributes: {
    label: {
      type: String,
    },
  },
};

const tabs = {
  render: "Tabs",
  attributes: {},
  transform(node, config) {
    const labels = node
      .transformChildren(config)
      .filter((child) => child && child.name === "Tab")
      .map((tab) => (typeof tab === "object" ? tab.attributes.label : null));

    return new Tag(this.render, { labels }, tabs);
  },
};

export default function Markdown({ post }) {
  const ast = Markdoc.parse(post.content);
  const finalAst = footnoteParse(ast);
  const content = Markdoc.transform(finalAst, {
    nodes: {
      fence,
      heading,
      footnoteItem,
      footnoteRef,
    },
    tags: {
      tabs,
      tab,
    },
  });
  return Markdoc.renderers.react(content, React, {
    components: {
      Fence,
      tabs: Tabs,
      tab: Tab,
    },
  });
}
```

components/Tab.js:

```js
import React from "react";
import { TabContext } from "./Tabs";

const Tab = ({ label, children }) => {
  const currentTab = React.useContext(TabContext);

  if (label !== currentTab) {
    return null;
  }

  return children;
};

export default Tab;
```

components/Tabs.js:

```js
import React from "react";

export const TabContext = React.createContext();

const Tabs = ({ labels, children }) => {
  const [currentTab, setCurrentTab] = React.useState(labels[0]);

  return (
    <TabContext.Provider value={currentTab}>
      <ul role="tablist">
        {labels.map((label) => (
          <li key={label}>
            <button
              role="tab"
              aria-selected={label === currentTab}
              onClick={() => setCurrentTab(label)}
            >
              {label}
            </button>
          </li>
        ))}
      </ul>
      {children}
    </TabContext.Provider>
  );
};

export default Tabs;
```

Any idea where it's trying to map some `children` variable that isn't defined?

**Comment by matildepark** [comment]
Ah, there we go. Thanks a lot.

**Comment by mfix-stripe** [comment]
Closed with ffcc159c94cf44ab2348bd18af4c46f3f7f6e0cb
